lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -329,6 +329,10 @@ def on_send(node)
             check_inflected(node)
           when :explicit
             check_explicit(node)
+          else
+            # :nocov:
+            raise ArgumentError("Unsupported style :#{style}")
```

Comment:
This is recurring. Even if RuboCop doesn't handle the inclusion validation of [SupportedStyles](https://github.com/rubocop/rubocop-rspec/blob/2be7260d25e78c740379ea56bd3c5ef0f8e21005/config/default.yml#L764) yet, we should open a ticket for it to do this.

Or do we still need those "else" clauses? Why?

Reply:
When we have a `case` or an `if` block without an `else` branch, it will _behave_ as if there was an empty branch:
```
else
  nil
end
```
SimpleCov know this, and marks the implicit `else` branch as not covered by tests. If we want to reach 100% branch coverage, we need to either change the last `elsif condition` into an `else`, or add an `else` branch.

I think you are right in the RuboCop itself will complain if a configured style isn’t in the listed available style options.

Reply:
I agree, and will shortly push up such changes.

However I think I’ll leave the two cases of `else; node.source` which seem like reasonable defaults for those case blocks.